src/wallet/rpcwallet.cpp

```diff
@@ -3338,8 +3346,8 @@ UniValue z_getbalance(const UniValue& params, bool fHelp)
         throw runtime_error(
             "z_getbalance \"address\" ( minconf )\n"
             "\nReturns the balance of a taddr or zaddr belonging to the node’s wallet.\n"
-            "\nCAUTION: If address is a watch-only zaddr, the returned balance may be larger than the actual balance,"
-            "\nbecause spends cannot be detected with incoming viewing keys.\n"
+            "\nCAUTION: If address is a watch-only Sprout shielded address, the returned balance may be larger"
+            "\nthan the actual balance, because spends cannot be detected with incoming viewing keys.\n"
```

Comment:
Sapling has incoming viewing keys too. The caution should be "If the wallet has only an incoming viewing key for this address, then spends cannot be detected and so the returned balance may be larger than the actual balance."